packages/gobba/gobba.0.4.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/gobba"
+bug-reports: "https://github.com/0x0f0f0f/gobba/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/gobba.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://0x0f0f0f.github.io/gobba"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["sh" "-c" "GOBBA_EXAMPLES=$(realpath ./examples/) dune runtest -p %{name}% -j %{jobs}%"] {with-test}
+  ["sh" "-c" "bisect-ppx-report -html coverage/ -I _build/default _build/default/test/bisect*.out"] {with-test}
```

Comment:
```suggestion
```
Arf sorry that's not working and needs rewiring (see https://github.com/ocaml/dune/issues/57), but also this makes no sense for opam users anyway. I think having only the `runtest` phase is good enough, my bad.